tcf_website/management/commands/load_semester.py

```diff
@@ -252,29 +252,49 @@ def load_section(
             units,
             section_type):
 
+        unique_params = {}
         params = {}
-        fields = {
+
+        unique_fields = {
             'sis_section_number',
             'semester',
+        }
+        fields = {
             'course',
             'topic',
             'units',
-            'section_type'}
+            'section_type'
+        }
+
         for k, v in locals().items():
```

Comment:
No idea, this was a preexisting part of the script. Any suggestions for how to fix it? (granted, it's probably outside the scope of this PR so we'd just have to refer back here)

Reply:
Maybe when you're done fixing the bugs, I can print `locals` to see what they are, but I have no clue at this point.

Reply:
Honestly I think since this is a script that gets run pretty much once a semester, it shouldn't matter too much if it's not exactly best practices if it works.